peeringdb_server/views.py

```diff
@@ -795,6 +803,18 @@ def view_profile_v1(request):
 
         data["networks"] = networks
 
+    # only add amr if amr scope is present
+    if scope_amr:
```

Comment:
What's the reason to make the field opt-in via one additional scope?

This claim is already defined optional in OpenID Core spec for the JWT ID token so implementations will not be surprised by is addition.

Reply:
@caguado The idea was to let the user decide if they want to share that data with the party requesting it. Do you disagree?

Reply:
I would disagree. From my interpretation of the [openid core spec](https://openid.net/specs/openid-connect-core-1_0.html#IDToken), this claim falls under the same core category as the subject, issuer, audience, issuance date, etc which are exposed with the `openid` scope and as they are part of the token attestation itself.